fetch.bs

```diff
@@ -5528,22 +5531,32 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
       <p>While true:
 
       <ol>
-       <li><p>If <var>timingInfo</var>'s
-       <a for="fetch timing info">final network-response start time</a> is 0, then set
-       <var>timingInfo</var>'s <a for="fetch timing info">final network-response start time</a> to
+       <li><p>Set <var>timingInfo</var>'s
+       <a for="fetch timing info">final network-response start time</a> to the
```

Comment:
So this will change for each response currently delivered with preceding 1xx responses, right? Does it not matter because this is not exposed CORS-cross-origin?

Reply:
Right, none of this is exposed when TAO fails, because we reconstruct a new `fetch timing info` with only start and end times.